File: VisitBosnia/VisitBosnia.Model/Requests/ReviewInsertRequest.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using System.Threading.Tasks;

namespace VisitBosnia.Model.Requests
{
    public class ReviewInsertRequest
    {

        public int AppUserId { get; set; }
        public int TouristFacilityId { get; set; }
        public int Rating { get; set; }
        public string? Text { get; set; }

    }
}

File: VisitBosnia/visit_bosnia_mobile/lib/model/review/review_insert_request.dart
import 'package:visit_bosnia_mobile/model/appUser/app_user.dart';

class ReviewInsertRequest {
  int? appUserId;
  int? touristFacilityId;
  int? rating;
  String? text;

  ReviewInsertRequest({
    this.appUserId,
    this.touristFacilityId,
    this.rating,
    this.text,
  });

  ReviewInsertRequest.fromJson(Map<String, dynamic> json) {
    appUserId = json['appUserId'];
    touristFacilityId = json['touristFacilityId'];
    rating = json['rating'];
    text = json['text'];
  }

  Map<String, dynamic> toJson() {
    final Map<String, dynamic> data = new Map<String, dynamic>();
    data['appUserId'] = this.appUserId;
    data['touristFacilityId'] = this.touristFacilityId;
    data['rating'] = this.rating;
    data['text'] = this.text;
    return data;
  }
}
